pep-0646.rst

```diff
@@ -873,7 +873,7 @@ Grammar Changes
 
 This PEP requires two grammar changes. Full diffs of ``python.gram``
 and simple tests to confirm correct behaviour are available at
-https://github.com/mrahtz/cpython/commits/pep646-grammar.
+https://github.com/mrahtz/cpython/commits/pep646-grammar-attempt2-rebased.
```

Comment:
Is that a tag or a branch? (GH makes it hard to tell.) Do you expect that it will still point to the same code 5 or 10 years from now? If not, I'm not sure it belongs in a PEP (which is a permanent artifact).

Reply:
It's a branch. But good point; I guess this information would be better conveyed to the SC in the cover letter, so I've removed it from the PEP.